setup.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+from setuptools import setup, find_packages
+
+from baseplate.integration.thrift.command import ThriftBuildPyCommand
+
+
+PY3 = (sys.version_info.major == 3)
+
+tests_require = [
+    "nose",
+    "coverage",
+    "webtest",
+]
+
+if not PY3:
+    tests_require.append("mock")
+
+setup(
+    name="reddit_service_ads_tracking",
+    packages=find_packages(),
+    install_requires=[
+        "pyramid",
+        "baseplate",
+    ],
+    cmdclass={
+        "build_py": ThriftBuildPyCommand,
+    },
+    entry_points={
+        "paste.app_factory": [
```

Comment:
Is this for `pshell` etc to work? Does it actually work? paste and baseplate have slightly different entry point requirements. Baseplate has some helpers for this purpose: https://github.com/reddit/baseplate/commit/3f6600ac35ade1dc5bfea8e81537b12888b618c9 . No setup.py needed, just a couple extra lines in your ini file.